tests/console/consoletest_setup.pm

```diff
@@ -13,6 +13,9 @@ use testapi;
 use utils;
 use strict;
 
+# Summary: console test pre setup, stoping and disabling packagekit, install curl and tar to get logs and so on
+# Maintainer: everybody@suse.de
```

Comment:
The maintainer is not the person that has to do everything but a person that can help if one can't understand himself. I don't mind if you add `Oliver Kurz <okurz@suse.de>` there